config/config.php

```diff
@@ -81,13 +81,21 @@
 	'production' => [
 	],
 	'stable' => [
+		'10.9' => [
+			'latest' => '10.9.1',
+			'web' => 'https://doc.owncloud.com/server/10.9/admin_manual/maintenance/upgrading/update.html',
+		],
+		'10.8' => [
+			'latest' => '10.9.1',
+			'web' => 'https://doc.owncloud.com/server/10.8/admin_manual/maintenance/upgrading/update.html',
+		],
```

Comment:
why was this section previously empty?

Reply:
@phil-davis because all channels were unified to deliver the same update path.
it was done by setting `	'eol_latest' => '10.7.100',` so 10.7.100 and below are using the same (EOL) channel.

Reply:
I mean they are using  EOL channel disregarding the requested channel